src/pybind/mgr/dashboard/frontend/src/app/shared/components/language-selector/supported-languages.enum.ts

```diff
@@ -15,12 +18,16 @@ export enum SupportedLanguages {
   'cs' = 'Čeština',
   'de-DE' = 'Deutsch',
   'en-US' = 'English',
+  'es-ES' = 'Español',
   'fr-FR' = 'Français',
   'id-ID' = 'Bahasa Indonesia',
+  'it-IT' = 'Italiano',
+  'ja-JP' = '日本語',
+  'ko-KR' = '한국어',
   'pl-PL' = 'Polski',
-  'pt-PT' = 'Português',
-  'es-ES' = 'Español',
-  'zh-CN' = '中文'
+  'pt-BR' = 'Português (brasileiro)',
```

Comment:
Just notice we are using this string for PT.
I would suggest just use `Português`, as there is no need to differentiate and we don't do it for other languages.
This should also be applied in master.
```suggestion
  'pt-BR' = 'Português',
```

Reply:
> This should also be applied in master.

Agreed. Can you please take care of this @Exotelis ?

Reply:
Removing the '(brasileiro)' is ok. 
However, I'd prefer to keep the language identifier as it is, since we had both languages on our project in the past. So on transifex we also have the pt_BR identifier and I'd like to keep it consistent.
If this is okay for you I'd just update the language text.